test/integration/list_constraints/launch.sh

```diff
@@ -13,63 +13,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#################################################################
-#   PLEASE FILL THE VARIABLES WITH VALID VALUES FOR TESTING     #
-#   DO NOT REMOVE/ADD ANY OF THE VARIABLES                      #
-#################################################################
-
-#################################################################
-#   RESOURCE IDENTIFIERS, VARIABLES AND ROLES                   #
-#################################################################
-
-# Project
-export PROJECT_ID="base-project-196723"
-
-# Organization
-export ORGANIZATION_ID="65779779009"
-
-# 1 above organization's folder to exclude from policy
-export FOLDER_EXCLUDE="886025180534"
-
-# Folder
-export FOLDER_ID="886025180534"
-
-# Folder 2 (must be different from folder above)
-export FOLDER_2_ID="560109990580"
-
-# 1 above organization's project to exclude from policy
-export PROJECT_EXCLUDE="base-project-196723"
-
-
-# Constraints
-
-# Applied to: project
```

Comment:
We shouldn't remove these.

Reply:
@morgante, absolutely! The constraints [should be](https://github.com/terraform-google-modules/terraform-google-org-policy/pull/18#discussion_r331296479) moved back.